reana_workflow_controller/app.py

```diff
@@ -42,6 +42,34 @@
 }
 
 
+@app.route('/workflows', methods=['GET'])
+def get_workflows():
+    """Get all workflows.
+    
+    .. http:get:: /workflows
+        Returns a JSON list with all the workflows.
```

Comment:
I would suggest to keep empty lines between different blocks when documenting as in `yadage_endpoint()` docstring, in favor of readability